File: member-service/src/main/java/com/hedvig/memberservice/aggregates/Member.java
package com.hedvig.memberservice.aggregates;

import lombok.Data;

import java.time.LocalDate;

@Data
public class Member {
  private String firstName;
  private String lastName;
  private String ssn;
  private String email;
  private String phoneNumber;

  private String acceptLanguage;

  private PickedLocale pickedLocale;

  private LivingAddress livingAddress;

  private LocalDate birthDate;
}

File: member-service/src/main/java/com/hedvig/memberservice/aggregates/Member.kt
package com.hedvig.memberservice.aggregates

import java.time.LocalDate

data class Member(
    val firstName: String? = null,
    val lastName: String? = null,
    val ssn: String? = null,
    val email: String? = null,
    val phoneNumber: String? = null,
    val acceptLanguage: String? = null,
    val pickedLocale: PickedLocale? = null,
    val livingAddress: LivingAddress? = null,
    val birthDate: LocalDate? = null
)
